### DISCUSSION TITLE
I made dyn-rules profiles

### DISCUSSION THREAD

**Discussion by Rudxain** [uBlockOrigin/uBlock-issues#3165] [Show and tell] [answered]
I made [uBO-R](https://github.com/Rudxain/uBO-rules) for me and my family/friends, then realized it could be useful for other people too, so I released it publicly.

`user.*` needs improvement and feedback. And I'm unsure about the repo's directory structure, as I have no idea how to organize it.

Feedback would be appreciated ❤️

**Comment by ghost** [comment, marked as answer]
It's not recommended to use someone else dynamic filtering rules and even to recommend advanced mode for casual users. You should create static filter lists for your users.

**Reply by Rudxain** [reply]
I totally agree. But the profiles are just meant to be templates. Although `user.*` aren't good enough as templates either
